exporter/opentelemetry-exporter-prometheus-remote-write/README.rst

```diff
@@ -1,27 +1,248 @@
-OpenTelemetry Prometheus Remote Write Exporter
-==============================================
+OpenTelemetry Python SDK Prometheus Remote Write Exporter
+=========================================================
 
-This library allows exporting metric data to `Prometheus Remote Write Integrated Backends
-<https://prometheus.io/docs/operating/integrations/>`_. Latest `types.proto
-<https://github.com/prometheus/prometheus/blob/master/prompb/types.proto>` and `remote.proto
-<https://github.com/prometheus/prometheus/blob/master/prompb/remote.proto>` Protocol Buffers 
-used to create WriteRequest objects were taken from Prometheus repository. Development is 
-currently in progress.
+This package contains an exporter to send `OTLP`_ metrics from the
+Python SDK directly to a Prometheus Remote Write integrated backend
+(such as Cortex or Thanos) without having to run an instance of the
+Prometheus server. The image below shows the two Prometheus exporters in
+the OpenTelemetry Python SDK.
+
+Pipeline 1 illustrates the setup required for a Prometheus "pull"
+exporter.
+
+Pipeline 2 illustrates the setup required for the Prometheus Remote
+Write exporter.
+
+|Prometheus SDK pipelines|
+
+The Prometheus Remote Write Exporter is a "push" based exporter and only
+works with the OpenTelemetry `push controller`_. The controller
+periodically collects data and passes it to the exporter. This exporter
+then converts the data into `timeseries`_ and sends it to the Remote
+Write integrated backend through HTTP POST requests. The metrics
+collection datapath is shown below:
+
+|controller_datapath_final|
+
+See the ``example`` folder for a demo usage of this exporter
+
+Table of Contents
+=================
+
+-  `Summary`_
+-  `Table of Contents`_
+
+   -  `Installation`_
+   -  `Quickstart`_
+   -  `Configuring the Exporter`_
+   -  `Securing the Exporter`_
+
+      -  `Authentication`_
+      -  `TLS`_
+
+   -  `Supported Aggregators`_
+   -  `Error Handling`_
+   -  `Retry Logic`_
+   -  `Contributing`_
+
+      -  `Design Doc`_
 
 Installation
 ------------
+Prerequisite
+~~~~~~~~~~~~
+1. Install the snappy c-library
+    **DEB**: `sudo apt-get install libsnappy-dev`
```

Comment:
Are there instructions for non-linux users?

Reply:
added instructions for windows